### ISSUE TITLE
(From TPW2) Player HQ Research items give no effect

### ISSUE THREAD

**Issue by Vectorial1024** [mewosmith/TeleportationWarsAlpha#1]
In Teleportation Wars 2, since the entire "Boso Ta" script is removed, even when the player completes a research item, no rewards are given.

Please re-implement the "research complete rewards" MD script, or find some other workable substitutes.

**Comment by mewosmith** [comment]
Yes, this is a bug.  I will leave this open so we do not forget the issue when we integrate PHQ.
